roles/secure-live-migration/tasks/passwordless-ssh.yaml

```diff
@@ -33,6 +33,7 @@
     state: present
     key: "{{ pf9_id_rsa.content | b64decode }}"
   when: item != ansible_fqdn
+  ignore_errors: true
```

Comment:
I set ignore_errors because it this step always fails.  I believe Josh is fixing this role in a separate branch.  So for now, can we leave this set to ignore_errors and the merge in Josh's changes later?

Reply:
The role typically fails because your hosts are not dns resolvable, this is a requirement for openstack cold migration in general.  Perhaps we should add a check for that?